data/human/pirate missions.txt

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2024 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+# Core Mining Concern Fleet
+mission "CMC Fleet"
+	invisible
+	landing
+	npc
+		on kill
+			set "dead edderkopp"
+		government "Pirate"
+		personality staying mining disables plunders harvests
+		system "Nashira"
+		ship "Mining Barge" "Edderkopp"
+	to complete
+		never
+	npc
+		government "Pirate"
+		personality mining disables
+		system "Nashira"
+		fleet
+			names "pirate"
+			variant
+				"Mining Drone" 4
+	npc
```

Comment:
Maybe move this bottom `npc` node to a new mission ("Pirates guarding the Edderkopp") that repeats so that the fleet can change periodically?